package.json

```diff
@@ -8,7 +8,10 @@
   },
   "author": "Engin Polat",
   "scripts": {
-    "pack": "npx tfx-cli extension create --rev-version"
+    "pack": "npx tfx-cli extension create --rev-version",
+    "pack-debug": "npx tfx-cli extension create --rev-version --overrides-file dev.json",
+    "web": "cd src && npx live-server --port=9393 --entry-file=main.html --host=localhost",
+    "debug": "npx live-server --port=33000 --entry-file=src/main.html --https=certs/https.js  --host=localhost"
```

Comment:
`web` and `debug` scripts are using two different approaches to run the `live-server`. `web` cd into the `src` folder and run it, `debug` runs it by providing the path of the `main.html`. can we make them similar?